### PR TITLE
Update Search API doc based on enum change

### PR DESCRIPTION
**Pull request by lanhongvp**
Update Documentation entityType namespace change

### CODE REVIEW

**Comment by hutingting1991** [comment]
Please update the PR name to be more specific, like "update entity type from microsoft.graph.message to message for search API"

File: api-reference/beta/resources/searchrequest.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The following is a JSON representation of the resource.
   "stored_fields": ["String"],
   "contentSources": ["String"],
   "entityTypes": ["String"],
-  "query": {"@odata.type": "microsoft.graph.searchQuery"},
```
Comment (hutingting1991) [inline api-reference/beta/resources/searchrequest.md:50]: revert this change

**Comment by CelesteDG** [comment]
Reviewed changes. Small fixes and no issues. Merging.

**Comment by CelesteDG** [comment]
Validation warning appeared after everything else previously was okay. Reverting merge to master to double check and fix the warning until build is clean.
